### PR TITLE
use slug as id for fetching collectibles

### PR DESCRIPTION
**Pull request by dpereskokov**
Fixes: #643 (1)

### CODE REVIEW

File: platform/ethereum/api_test.go [hunk 1]
```diff
@@ -431,8 +431,8 @@ var collection2Dst = blockatlas.Collection{
 	Description:     "Year 4424: The search begins for new life on the other side of the galaxy. Explore abandoned space stations, mysterious caverns, and ruins on far away worlds in order to unlock puzzles and secrets! Beware the rogue machines!",
 	ExternalLink:    "https://www.ageofrust.games/",
 	Total:           1,
-	CategoryAddress: "0xfaafdc07907ff5120a76b34b731b278c38d6043c---age-of-rust",
-	Id:              "0xfaafdc07907ff5120a76b34b731b278c38d6043c---age-of-rust",
+	CategoryAddress: "0xfaafdc07907ff5120a76b34b731b278c38d6043c",
```
Comment (vikmeup) [inline platform/ethereum/api_test.go]: `ID` and `CategoryAddress` should be the same value and `CategoryAddress ` will be removed in the future

**Comment by Pantani** [comment]
the tests break

**Comment by vikmeup** [comment]
@dpereskokov can you rebase this? I think it might be ok to merge this. can you deploy to staging to test this out?

**Comment by vikmeup** [comment]
@dpereskokov any updates on this?

**Comment by dpereskokov** [comment]
This PR should be reopened since mobiles store collectables with old id in db and there will be duplicates

**Pull request merged**